README.md

```diff
@@ -12,3 +44,37 @@ Provide a detailed description and offer tools to implement product passports.
 
 We have an reference implementation which is the Digital Product Pass.
 You find the repository [here](https://github.com/eclipse-tractusx/digital-product-pass)
+
+### Documentation
+
+To get started you can have a look into our documentation:
+
+| Name                                                                      | Description                                                                                                                                                 |
+| ----------------------------------------------------------------          |-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Arc42](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/arc42/Arc42.md)                                             | Main Architecture Document (Arc42) of Digital Product Pass Application                                                                                      |
```

Comment:
(Arc42) of **the** Digital Product Pass Application

Reply:
Fixed